src/Compilers/Test/Core/TestHelpers.cs

```diff
@@ -163,7 +163,7 @@ public static ImmutableArray<byte> HexToByte(ReadOnlySpan<char> input)
 
             return ImmutableCollectionsMarshal.AsImmutableArray(bytes);
 
-            byte parseByte(ReadOnlySpan<char> input, NumberStyles numberStyle)
+            static byte parseByte(ReadOnlySpan<char> input, NumberStyles numberStyle)
```

Comment:
I don't have any object, but this change (and the one in Silverlight.cs) are unexpected in this PR. Just want to confirm these were intentional

Reply:
It was intentional. It's a follow up comment from a previous PR that I forgot to merge.